### PR TITLE
added requirement for admin login

### PR DESCRIPTION
**Pull request by JordanHartley**
## Change Summary
Login is now required to use the queue

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [Y ] The pull request title has an issue number
- [ Y] The code works by "Smoke testing" (in other words, you have tested it your own and it works)
- [ Y] The code has documentation (either comments or in main documentation platform)

## Other Information
The username and password will be whatever is in your local .env file

### CODE REVIEW

File: app/app/__init__.py [hunk 1]
```diff
@@ -26,6 +27,7 @@
 from app.routes import routes
 from app.queue import queue
 
-app.register_blueprint(routes, url_prefix='/')
-app.register_blueprint(auth, url_prefix='/')
-app.register_blueprint(queue, url_prefix='/')
\ No newline at end of file
+app.register_blueprint(routes, url_prefix="/")
+app.register_blueprint(auth, url_prefix="/")
+app.register_blueprint(queue, url_prefix="/")
```
Comment (TheOneHoff) [inline app/app/__init__.py:32]: Seems fine

File: app/app/templates/login.html [hunk 2]
```diff
@@ -1,22 +1,67 @@
...
+			<div class="card-footer justify-content-center">
+			  <input type="submit" class="btn btn-info btn-link btn-lg" name="" value="Login" />
+			</div>
+		  </div>
+		</form>
+	  </div>
+	</div>
+  </div>
+</div>
+{% endblock %}
```
Comment (TheOneHoff) [inline app/app/templates/login.html:67]: Login page looks great!

File: app/app/auth.py [hunk 3]
```diff
@@ -2,14 +2,31 @@
...
+    new_user.set_password(admin_password)
+    db.session.add(new_user)
+    db.session.commit()
+
 @auth.route('/login', methods=['GET', 'POST'])
 def admin_login():
+    admin = User.query.filter_by(username=admin_username).first()
+    # If admin name/password changed delete previous admin user 
+    if (admin and not admin.check_password(admin_password)) or not admin:
+        changed_admin_login()
```
Comment (TheOneHoff) [inline app/app/auth.py:28]: Looks good to me

**Comment by Alexkszzz** [comment]
All looks great for me!

**Review by TheOneHoff — APPROVED** [review]
Great stuff. Authentication works in my testing. I can't see any immediate problems

**Pull request merged**